HISTORY.md

```diff
@@ -1,13 +1,5 @@
 # Release history
 
-## 0.10.0 (2022-07-31)
-### Features
-- Add validation for form data
-- Handle extra headers returned by route functions
-
-### Internal
-- Cleanup pipelines, drop python 3.6 tests, test on MacOS images
-
```

Comment:
Can we remove this diff? You're probably not in sync with the current master branch